src/scanner.js

```diff
@@ -19,40 +17,37 @@ async function scanner(cfg, authorizer) {
   // for scans to take longer than for the auth service to be overloaded.
   let auth = new taskcluster.Auth({credentials: cfg.app.credentials});
 
-  // gather all identityProviderIds used in any authorizer
-  let identityProviders = authorizer.identityProviders;
-
-  // enumerate all clients for any of those identity providers
-  let clients = [];
-  for (let idp of identityProviders) {
-    clients = clients.concat(await auth.listClients({prefix: idp + '/'}));
-  }
-
-  // sort by clientId, so that each identity (a prefix of the clientId) appears
-  // contiguously
-  clients = _.sortBy(clients, 'clientId');
+  const clients = await auth.listClients();
```

Comment:
I think we can still use a prefix for this search -- `mozilla-auth0/`

Reply:
We should probably abstract that out to be a part of the handler, but we can do that later.